### PR TITLE
create a footer component for new branch

### PR DESCRIPTION
**Pull request by kidus101**
Pull request for latest pull request

### CODE REVIEW

File: Web/starter_project_barbies/components/layout/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+
+const Footer = () => {
```
Comment (sam5235) [inline Web/starter_project_barbies/components/layout/Footer.tsx:6]: add this for formality - React.FC

File: Web/starter_project_barbies/components/layout/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        </div>
+
+{/* Footer Coyright  */}
+        <div className="sm:flex sm:items-center sm:justify-between">
+          <span className="text-sm text-black sm:text-center dark:text-black">©  <a href="https://flowbite.com/" className="hover:underline">2020 Africa To silicon Valley.Inc</a>. All Rights Reserved.
+          </span>
+          <div className="flex mt-4 space-x-6 pr-4 pb-6 pt-2 sm:justify-center sm:mt-0">
+
+              <Link href="#" className="text-black hover:text-gray-900 dark:hover:text-white">
+                  <svg className="w-5 h-5" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true"><path d="M8.29 20.251c7.547 0 11.675-6.253 11.675-11.675 0-.178 0-.355-.012-.53A8.348 8.348 0 0022 5.92a8.19 8.19 0 01-2.357.646 4.118 4.118 0 001.804-2.27 8.224 8.224 0 01-2.605.996 4.107 4.107 0 00-6.993 3.743 11.65 11.65 0 01-8.457-4.287 4.106 4.106 0 001.27 5.477A4.072 4.072 0 012.8 9.713v.052a4.105 4.105 0 003.292 4.022 4.095 4.095 0 01-1.853.07 4.108 4.108 0 003.834 2.85A8.233 8.233 0 012 18.407a11.616 11.616 0 006.29 1.84" /></svg>
```
Comment (sam5235) [inline Web/starter_project_barbies/components/layout/Footer.tsx:91]: put this string somewhere on variable.

**Review by sam5235 — CHANGES_REQUESTED** [review]
fix the comments

File: Web/starter_project_barbies/components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import Link from "next/link";
+import Image from "next/image";
+
+
+const Footer:React.FC = () => {
+  return (
+    <footer className="bg-white border-black p-4">
+      <div className="flex justify-between mt-4">
+
+      <div className="flex items-center">
```
Comment (nawrazi) [inline Web/starter_project_barbies/components/layout/Footer.tsx:11]: Indentations seem off, I would suggest running lint.

**Review by nawrazi — CHANGES_REQUESTED** [review]
Looks good, I have left one comment.

**Pull request closed without merging**